### ISSUE TITLE
Default use-case doesn't seem to be covered.

### ISSUE THREAD

**Issue by ysig** [mgedmin/findimports#23]
Correct me if I'm wrong but I feel that this package doesn't easily cover a default use case which is:

> Given I have worked on a project for an amount of time and I have installed different external packages:
> - run the package locally and display which external packages are really needed
> - optionally produce a requirements file 
> - optionally add current version as from pip show

Thank you for this package,

**Comment by mgedmin** [comment]
That is correct.  findimports is older than setuptools and doesn't know anything about that side of things.
